plugins/synaptics-rmi/fu-synaptics-rmi-device.c

```diff
@@ -178,6 +178,13 @@ fu_synaptics_rmi_device_set_page (FuSynapticsRmiDevice *self, guint8 page, GErro
 	return TRUE;
 }
 
+void		 
+fu_synaptics_rmi_device_set_iepmode	(FuSynapticsRmiDevice *self, gboolean iepmode)
```

Comment:
nitpick: just one space between the function and the bracket, then LGTM, thanks.